test/index_test.js

```diff
@@ -64,6 +65,12 @@ describe("Kinto", () => {
         expect(Kinto.transformers.RemoteTransformer).to.eql(RemoteTransformer);
       });
     });
+
+    describe("get syncSrategy()", () => {
+      it("should provide an transformers static getter", () => {
```

Comment:
Typo "syncSrategy" -> "syncStrategy" (one line up) and "an transformers" -> "a syncStrategy"